cmd/policy-assistant/pkg/matcher/traffic.go

```diff
@@ -71,10 +75,245 @@ func (p *TrafficPeer) IsExternal() bool {
+	if len(workloadMetadata) != 3 || (workloadMetadata[0] == "" || workloadMetadata[1] == "" || workloadMetadata[2] == "") || (workloadMetadata[1] != "daemonset" && workloadMetadata[1] != "statefulset" && workloadMetadata[1] != "replicaset" && workloadMetadata[1] != "deployment" && workloadMetadata[1] != "pod") {
+		logrus.Fatalf("Bad Workload structure: Types supported are pod, replicaset, deployment, daemonset, statefulset, and 3 fields are required with this structure, <namespace>/<workloadType>/<workloadName>")
+	}
+	kubeClient, err := kube.NewKubernetesForContext("")
+	utils.DoOrDie(err)
+	ns, err := kubeClient.GetNamespace(workloadMetadata[0])
+	utils.DoOrDie(err)
+	kubePods, err := kube.GetPodsInNamespaces(kubeClient, []string{workloadMetadata[0]})
+	if err != nil {
+		logrus.Fatalf("unable to read pods from kube, ns '%s': %+v", workloadMetadata[0], err)
+	}
+	for _, pod := range kubePods {
+		if workloadMetadata[1] == "deployment" && pod.OwnerReferences[0].Kind == "ReplicaSet" {
+			kubeReplicaSets, err := kubeClient.GetReplicaSet(workloadMetadata[0], pod.OwnerReferences[0].Name)
+			if err != nil {
+				logrus.Fatalf("unable to read Replicaset from kube, rs '%s': %+v", pod.OwnerReferences[0].Name, err)
+			}
+			workloadOwner = kubeReplicaSets.OwnerReferences[0].Name
+		} else if workloadMetadata[1] == "daemonset" || workloadMetadata[1] == "statefulset" || workloadMetadata[1] == "replicaset" {
+			workloadOwner = pod.OwnerReferences[0].Name
+		} else if workloadMetadata[1] == "pod" {
+			workloadOwner = pod.Name
+		}
+		if workloadOwner == workloadMetadata[2] {
+			podLabels = pod.Labels
+			namespaceLabels = ns.Labels
+			podNetworking := PodNetworking{
+				IP: pod.Status.PodIP,
+			}
+			podsNetworking = append(podsNetworking, &podNetworking)
+			workloadOwnerExists = true
+
+		}
+	}
+
+	if !workloadOwnerExists {
+		logrus.Infof("workload not found on the cluster")
+		internalPeer = InternalPeer{
+			Workload: "",
+		}
+	} else {
+		internalPeer = InternalPeer{
+			Workload:        p.Internal.Workload,
+			PodLabels:       podLabels,
+			NamespaceLabels: namespaceLabels,
+			Namespace:       workloadMetadata[0],
+			Pods:            podsNetworking,
+		}
+	}
+
+	TranslatedPeer := TrafficPeer{
+		Internal: &internalPeer,
+	}
+	return TranslatedPeer
+}
+
+func DeploymentsToTrafficPeers() []TrafficPeer {
+	var deploymentPeers []TrafficPeer
+	kubeClient, err := kube.NewKubernetesForContext("")
+	utils.DoOrDie(err)
+	kubeNamespaces, err := kubeClient.GetAllNamespaces()
+	if err != nil {
+		logrus.Fatalf("unable to read namespaces from kube: %+v", err)
+	}
+
+	for _, namespace := range kubeNamespaces.Items {
+		kubeDeployments, err := kubeClient.GetDeploymentsInNamespace(namespace.Name)
+		if err != nil {
+			logrus.Fatalf("unable to read deployments from kube, ns '%s': %+v", namespace.Name, err)
+		}
+		for _, deployment := range kubeDeployments {
+			TmpInternalPeer := InternalPeer{
```

Comment:
nit: [conventional in go](https://medium.com/@singhalok641/mastering-exported-and-unexported-names-in-go-a-key-to-effective-encapsulation-6d4b28bd61a6) to use lowercase for variables in functions